v6/scripts/signFiles.sh

```
@@ -0,0 +1,3 @@
+for f in release/*; do
```

Shellcheck

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)